### PR TITLE
feat: support rpx in ssr

### PR DESCRIPTION
**Pull request by chenjun1011**
*Before* submitting a pull request, please make sure the following is done...

1. Fork the repo and create your branch from `master`.
2. If you've added code that should be tested, add tests!
3. If you've changed APIs, update the documentation.
4. Ensure the test suite passes (`npm test`).
5. Make sure your code lints (`npm run lint`) - we've done our best to make sure these rules match our internal linting guidelines.

### CODE REVIEW

File: packages/rax-server-renderer/src/index.js [hunk 1]
```diff
@@ -115,6 +113,19 @@ function styleToCSS(style, options = {}) {
   return css;
 }
 
+function rpx2vw(val, opts) {
+  const pixels = parseFloat(val);
+  const parsedVal = toFixed(pixels / opts.viewportWidth * 100, opts.unitPrecision);
+  return parsedVal;
+}
+
+function toFixed(number, precision) {
```
Comment (Orange-C) [inline packages/rax-server-renderer/src/index.js]: 为什么不用js的toFixed

Comment (chenjun1011) [reply packages/rax-server-renderer/src/index.js]: var t = 5;  t.toFixed(5)  输出 5.00000
而通过上面的方法  toFixed(5)  输出 5

File: packages/rax-scripts/package.json [hunk 2]
```diff
@@ -75,6 +75,7 @@
     "postcss": "^7.0.17",
     "postcss-import": "^12.0.1",
     "postcss-loader": "^3.0.0",
+    "postcss-plugin-rpx2vw": "^0.0.1",
```
Comment (boiawang) [inline packages/rax-scripts/package.json]: https://github.com/alibaba/rax/pull/1146 这个 pr 已经加了

File: packages/rax-scripts/src/config/webapp/webpack.config.base.js [hunk 3]
```diff
@@ -75,6 +75,7 @@ module.exports = function getWebpackBaseConfig(options = {}) {
                     },
                     stage: 3,
                   }),
+                  require('postcss-plugin-rpx2vw')
```
Comment (boiawang) [inline packages/rax-scripts/src/config/webapp/webpack.config.base.js]: 同上

File: packages/rax-server-renderer/src/__tests__/renderToString.js [hunk 4]
```diff
@@ -65,6 +65,67 @@ describe('renderToString', () => {
...
+      flex: 1,
+      fontSize: '16rpx',
+      width: '100%'
+    };
+
+    function MyComponent(props, context) {
+      return <div style={style} />;
+    }
+
+    let str = renderToString(<MyComponent />);
```
Comment (boiawang) [inline packages/rax-server-renderer/src/__tests__/renderToString.js]: let => const

File: packages/rax-server-renderer/src/__tests__/renderToString.js [hunk 5]
```diff
@@ -65,6 +65,67 @@ describe('renderToString', () => {
...
+
+    function MyComponent(props, context) {
+      return (
+        <div style={styles.container}>
+          <p style={styles.text}>Hello</p>
+        </div>
+      );
+    }
+
+    let str = renderToString(<MyComponent />);
```
Comment (boiawang) [inline packages/rax-server-renderer/src/__tests__/renderToString.js]: let => const

File: packages/rax-server-renderer/src/__tests__/renderToString.js [hunk 6]
```diff
@@ -65,6 +65,67 @@ describe('renderToString', () => {
...
+
+    function MyComponent(props, context) {
+      return (
+        <div style={styles.container}>
+          <p style={styles.text}>Hello</p>
+        </div>
+      );
+    }
+
+    let str = renderToString(<MyComponent />, {
```
Comment (boiawang) [inline packages/rax-server-renderer/src/__tests__/renderToString.js]: let => const

File: packages/rax-server-renderer/src/index.js [hunk 7]
```diff
@@ -82,31 +82,31 @@ function merge(target) {
   return target;
 }
 
+const DEFAULT_STYLE_OPTIONS = {
+  defaultUnit: 'px',
```
Comment (boiawang) [inline packages/rax-server-renderer/src/index.js]: defaultUnit 不是 rpx 吗？

Comment (chenjun1011) [reply packages/rax-server-renderer/src/index.js]: 不是  web 标准 应该是 px

**Pull request merged**